File: app/src/main/java/com/digicoffer/lauditor/Members/MembersModel.java
package com.digicoffer.lauditor.Members;

import org.json.JSONArray;

public class MembersModel {
    String currency;
    String defaultRate;
    String designation;
    String email;
    String id;
    String lastLogin;
    String name;
    JSONArray groups;
    String group_id;
    String group_name;
    boolean isdisabled;

    public boolean isIsdisabled() {
        return isdisabled;
    }

    public void setIsdisabled(boolean isdisabled) {
        this.isdisabled = isdisabled;
    }

    public String getGroup_id() {
        return group_id;
    }

    public void setGroup_id(String group_id) {
        this.group_id = group_id;
    }

    public String getGroup_name() {
        return group_name;
    }

    public void setGroup_name(String group_name) {
        this.group_name = group_name;
    }

    public String getCurrency() {
        return currency;
    }

    public void setCurrency(String currency) {
        this.currency = currency;
    }

    public String getDefaultRate() {
        return defaultRate;
    }

    public void setDefaultRate(String defaultRate) {
        this.defaultRate = defaultRate;
    }

    public String getDesignation() {
        return designation;
    }

    public void setDesignation(String designation) {
        this.designation = designation;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getLastLogin() {
        return lastLogin;
    }

    public void setLastLogin(String lastLogin) {
        this.lastLogin = lastLogin;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public JSONArray getGroups() {
        return groups;
    }

    public void setGroups(JSONArray groups) {
        this.groups = groups;
    }


}

File: app/src/main/java/com/digicoffer/lauditor/Members/MembersModel.kt
package com.digicoffer.lauditor.Members

import org.json.JSONArray

class MembersModel {
    var currency: String? = null
    var defaultRate: String? = null
    var designation: String? = null
    var email: String? = null
    var mobile: String? = null
    var phone: String? = null
    var contact_phone: String? = null
    var phone_number: String? = null
    var id: String? = null
    var lastLogin: String? = null
    var name: String? = null
    var groups: JSONArray? = null
    var group_id: String? = null
    var group_name: String? = null

    @get:JvmName("isIsdisabled")
    @set:JvmName("setIsdisabled")
    var isdisabled: Boolean = false
}
